# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/tree/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

The usual gcc-3.4 fixes.

  - Rene Rebe <rene@exactcode.de>
  - Andreas V. Meier <avmeier@web.de>

diff -ruN tree-1.4/tree.c tree-1.4-new/tree.c
--- tree-1.4/tree.c	2003-02-06 17:24:11.000000000 +0100
+++ tree-1.4-new/tree.c	2004-05-15 22:09:13.994582920 +0200
@@ -1083,6 +1083,7 @@
       case '?':
       case '+':
       default:
+      ((void)0);
     }
   }
 }
